tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -51,6 +55,10 @@ When('the user browses to the shared-with-others page', function() {
   return client.page.sharedWithOthersPage().navigateAndWaitTillLoaded()
 })
 
+When('the user browses to the shared-by-link page', function() {
+  return client.page.sharedViaLinkPage().navigateAndWaitTillLoaded()
```

Comment:
the logic is same as in `Given('the user has browsed to the shared-by-link page', function() {` so may be you can reuse it.

Reply:
imo for a when step `.navigateToUsingMenu('Shared via link')` will be better rather than navigateAndWaitTillLoaded because verification will not be necessary

Reply:
found this step `the user browses to the shared-via-link page using the webUI`